index.html

```diff
@@ -36,7 +36,7 @@
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
           </button>
-          <a class="navbar-brand" href="#">Project name</a>
+          <a class="navbar-brand" href="#">PDG Poetry Project</a>
```

Comment:
It is important for our poetry website's name to be on the navbar. This branch passes and I endorse merge.

Reply:
I was unable to really get anything running in my environment, but I agree with Dineen's changes.